services/s3/config/crd/bases/s3.services.k8s.aws_buckets.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.3.1-0.20200716001835-4a903ddb7005
+    controller-gen.kubebuilder.io/version: (devel)
```

Comment:
hmm.... not sure this is correct.

Reply:
My bad, i was used the latest controller-gen version, i just regenerated with `v3.1.0`